README.md

```diff
@@ -2,9 +2,13 @@
 
 ## What's in this repo
 
-The govuk-infrastructure repo contains Terraform modules for turning up an EKS Kubernetes cluster for GOV.UK.
+The govuk-infrastructure repo contains:
 
-Also included are reusable workflows for running the code scanning tools Rubocop, Brakeman and the CodeQL Static Application Security Tests (SAST).
+- [`terraform/`](terraform/): Terraform modules for turning up an Kubernetes
+  cluster on EKS for GOV.UK.
+- [`.github/`](.github/): GitHub Actions and workflows used by other GOV.UK
+  repos, for example release automation, test runners and security analysis
+  tools.
```

Comment:
Dunno if we also wanna mention it has the Dockerfile/responsible for the build of the toolbox image: https://github.com/alphagov/govuk-infrastructure/blob/main/images/toolbox/Dockerfile

Reply:
Ooh good catch - thanks!

Reply:
Done :)